src/APL/saleor-cloud-apl.ts

```diff
@@ -132,10 +139,13 @@ export class SaleorCloudAPL implements APL {
       });
 
       debug(`Delete responded with ${response.status} code`);
-    } catch (error) {
-      debug("Error during deleting the data: %s", error);
+    } catch (error: any) {
```

Comment:
```
  } catch (error) {
    debug("error");
    let message = "Unknown error"
    if (error instanceof Error) {
      message = error.message;
    }
```

WDYT?

Reply:
I can check `"message" in error`, but we can't rely on `instanceof` because in js you can throw anything.

I will make it without `any`

Reply:
Refactored